docs/doc-guideline.md

```diff
@@ -0,0 +1,52 @@
+# DOC GUIDELINE
+
+## How to write an article for the website
+
+1. All the problems written should necessarily include 3 sections
+   1. Explanation of the article
+   2. Sample input
+   3. Sample output
```

Comment:
I guess we can make it six sections: 

- Introduction about the Algorithm/Data Structure
- Explanation about the Algorithm/Data Structure
- Algorithmic Steps for implementation
- Code (C++/Java and Python as of now) 
- Input-Output Samples (Atleast two) 
- Time & Space Complexity 

This is my rough idea and a lot of people are following this. Make them descriptive and presentable

Reply:
This sounds elaborate. I will make the changes